src/ngraph/op/variadic_split.cpp

```diff
@@ -53,31 +59,62 @@ void ngraph::op::v1::VariadicSplit::validate_and_infer_types()
-
-            auto splits_length = std::accumulate(split_lengths.begin(), split_lengths.end(), 0UL);
-
-            NODE_VALIDATION_CHECK(this, axis > 0, "Provided axis:", axis, " can not be negative");
             auto data_rank = static_cast<size_t>(data_shape.rank());
+            auto axis_val = as_type_ptr<op::Constant>(axis_input)->get_vector<int64_t>()[0];
+
+            // Adjust split axis in case of negatives
+            int64_t axis = (axis_val < 0) ? data_rank + axis_val : axis_val;
             NODE_VALIDATION_CHECK(this,
-                                  axis < data_rank,
-                                  "Provided axis:",
-                                  axis,
-                                  " can not be higher than input data rank: ",
+                                  axis >= 0 && axis < data_rank,
+                                  "Provided axis: ",
+                                  axis_val,
+                                  " exceeds input data rank: ",
                                   data_rank);
 
+            auto split_lengths =
+                as_type_ptr<op::Constant>(split_lengths_input)->get_vector<int64_t>();
+            // Adjust split lengths in case of negatives
+            size_t sum_of_splits = 0;
+            int64_t negative_one = -1;
+            for (size_t i = 0; i < split_lengths.size(); i++)
+            {
+                NODE_VALIDATION_CHECK(this,
+                                      split_lengths[i] >= -1,
+                                      "Invalid value ",
+                                      split_lengths[i],
+                                      " in split lengths input. Should be >= -1.");
+
+                if (split_lengths[i] == -1)
+                {
+                    NODE_VALIDATION_CHECK(this,
+                                          negative_one == -1,
+                                          "Cannot infer split with multiple -1 values at ",
+                                          negative_one,
+                                          " and ",
+                                          i);
+                    negative_one = i;
+                }
+                else
+                {
+                    sum_of_splits += split_lengths[i];
+                }
+            }
+
             NODE_VALIDATION_CHECK(this,
-                                  splits_length == static_cast<size_t>(data_shape[axis]),
-                                  "Total length of splits:",
-                                  splits_length,
-                                  " does not sum to length of the choosen axis: ",
+                                  sum_of_splits <= static_cast<size_t>(data_shape[axis]),
+                                  "Total length of splits: ",
+                                  sum_of_splits,
+                                  " exceeds the length of the chosen axis: ",
                                   static_cast<size_t>(data_shape[axis]));
```

Comment:
In the case when we don't have `the nagative one` this check will pass an invalid parameters.
example:
```
axis = 1
data_shape = {2, 3, 4, 5}
split_lengths = {1, 1}
```